package.json

```diff
@@ -63,5 +63,8 @@
   },
   "resolutions": {
     "@types/react": "^16.8.17"
+  },
+  "dependencies": {
+    "browser-headers": "^0.4.1"
```

Comment:
I'm not sure if what we're doing is correct or not, but currently all of our deps go up in dev dependencies lol

Reply:
woops, moved this up. IIUC it doesn't really matter which deps section we use in our case because we aren't publishing an NPM package